components/Admissions/EventItem.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import moment from "moment";
+import { Box, Flex, Text, Image, Link } from "native-base";
+
+function EventItem(
+  id: number,
+  title: string,
+  cover: string,
+  location: string,
+  eventType: string,
+  startTime: string
+) {
+  interface EventType {
+    [key: string]: string;
+  }
+
+  const EVENT_CONSTANTS: EventType = {
+    company_presentation: "Bedriftspresentasjon",
+    lunch_presentation: "Lunsjpresentasjon",
+    alternative_presentation: "Alternativ bedpres",
+    course: "Kurs",
+    breakfast_talk: "Frokostforedrag",
+    party: "Fest",
+    social: "Sosialt",
+    event: "Arrangement",
+    kid_event: "KID-arrangement",
+    other: "Annet",
+  };
+
+  const eventTypeToString = (eventT: string) => {
+    return EVENT_CONSTANTS[eventT] || EVENT_CONSTANTS.other;
+  };
```

Comment:
These should be moved outside of the EventItem component, preferably inside a types.ts file somewhere in the repo. I suggest adding a folder called /lib/types.ts (in the repo root folder)

Reply:
I fixed. Thanks for the feedback.